entry

```diff
@@ -1,29 +1,83 @@
 #!/bin/sh
-set -e -x
+set -ex
 
 trap exit TERM INT
 
-for src_range in ${SRC_RANGES}; do
+info()
+{
+    echo '[INFO] ' "$@"
+}
+fatal()
+{
+    echo '[ERROR] ' "$@" >&2
+    exit 1
+}
+
+check_iptables_mode() {
+    lsmod | grep "nf_tables" 2> /dev/null
+    if [ $? = 0 ]; then
+        mode=nft
+    else 
+        mode=legacy
+    fi
+}
+
+set_nft() {
+    bin_dir="/sbin"
+
+    for i in iptables iptables-save iptables-restore ip6tables; do 
+        ln -sf "xtables-nft-multi" "$base_path/$i"; 
```

Comment:
Should this be `bin_dir` not `base_path`? I don't see `base_path` defined anywhere.
```suggestion
        ln -sf "xtables-nft-multi" "$bin_dir/$i"; 
```

Reply:
yeah its a typo, fixing now